graphbolt/src/fused_csc_sampling_graph.cc

```diff
@@ -1031,9 +1038,14 @@ torch::Tensor TemporalMask(
     auto neighbor_timestamp =
         node_timestamp.value().index_select(0, csc_indices.slice(0, l, r));
     mask &= neighbor_timestamp < seed_timestamp;
+    if (seed_time_window.has_value())
+      mask &= neighbor_timestamp > seed_timestamp - seed_time_window.value();
```

Comment:
Are we assuming the the indices are sorted when we apply windows?

Reply:
No, it is not.